src/style.css

```diff
@@ -0,0 +1,147 @@
+* {
```

Comment:
Please create separate file for each BEM block, as asked in 10 item in checklist&.